README.md

```diff
@@ -15,22 +20,24 @@
 
 ## A Free GIS package for Greenland
```

Comment:
We describe what "QGreenland" is here, but not `qgreenland`. Maybe put some text before this section about this repo? E.g.,

> This repository is responsible for the code and configuration for creating the QGreenland Core zip package. To download the package and learn more about QGreenland, see ...